app/serializers/recipe_serializer.rb

```diff
@@ -1,24 +1,20 @@
 class RecipeSerializer < ActiveModel::Serializer
-  attributes :id, :name, :content, :video_link, :cooking_time, :price, :level, :rates
-  
+  attributes :id, :name, :content, :price, :video_link, :cooking_time, :price, :comments, :level, :rates
```

Comment:
```suggestion
  attributes :id, :name, :content, :price, :video_link, :cooking_time, :level, :comments, :rates
```
You had duplicated attribute "price".